bin/MadGraph5_aMCatNLO/cards/production/13TeV/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM/WBJetsToLNu_Wpt-80toInf_5f_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,21 @@
+import model sm-ckm_no_b_mass
+
+define p = p b b~
+define j = p
+define bquark = b b~
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+# Specify process(es) to run
+generate p p > ell+ vl bquark / h @0
+add process p p > ell+ vl bquark j / h @1
+add process p p > ell+ vl bquark j j j / h @2
```

Comment:
these are "out of order" which might cause confusion downstream when looking at the matching efficiency tables